general/development/process/release/index.md

```diff
@@ -168,6 +168,7 @@ Usually on Monday
 | 5. | &#10003; | &#10003; | Notify about publications of CVE using form: https://cveform.mitre.org/ | Security Officer |
 | 6. | &#10003; |  | Upgrade moodle.org and all other Moodle community sites (next sites first, then production) |  |
 | 7. | &#10003; |  | **Deprecations:**<ul><li>If not yet present, create the deprecation epic for the next series (`X+1.0`) similar to [MDL-79719](https://tracker.moodle.org/browse/MDL-79719). The issue can be based/cloned from the previous deprecation epic and linked ("has to be done after") with it.</li><li>Add standard issues to it:<ul><li>Final deprecation of `lib/deprecatedlib.php` (like [MDL-79720](https://tracker.moodle.org/browse/MDL-79720))</li><li>Removal of deprecated Behat steps (like [MDL-79721](https://tracker.moodle.org/browse/MDL-79721))</li><li>Removal of strings (like [MDL-78078](https://tracker.moodle.org/browse/MDL-79722)) and</li><li>Removal of renamed classes ([MDL-79723](https://tracker.moodle.org/browse/MDL-79723)).</li></ul></li><li>Review [how previous deprecation epics are going](https://tracker.moodle.org/issues/?jql=project%20%3D%20MDL%20AND%20text%20~%20%22Collect%20together%20deprecated%20and%20planned%20code%20changes%20for%20Moodle%22%20AND%20affectedVersion%20in%20(releasedVersions()%2C%20%22Future%20Dev%22)%20AND%20resolution%20%3D%20Unresolved%20ORDER%20BY%20issuekey%20%20DESC) and share about them.</li></ul><br/>**Libraries:**<ul><li>Create the next third party libraries epic for `X.Y+1`, similar to [MDL-81653](https://tracker.moodle.org/browse/MDL-81653). The issue should refer to the list of [Moodle libraries credits](../../../community/credits/thirdpartylibs.md). The issue can be based/cloned on the previous one and linked ("has to be done after") with it. **Drag/link opened issues from the previous one**, avoiding leaving unfinished stuff.</li><li>Review [how previous library review epics are going](https://tracker.moodle.org/issues/?jql=project%20%3D%20MDL%20AND%20(summary%20~%20%22Review%20third%20party%20libraries%20for%20new%20versions%22%20OR%20summary%20~%20%22Review%20third-party%20libraries%20for%20upgrade%22)%20AND%20affectedVersion%20in%20(releasedVersions()%2C%20%22Future%20Dev%22)%20AND%20resolution%20%3D%20Unresolved%20ORDER%20BY%20issuekey%20%20DESC) and share about them.</li></ul> | Integration Team |
+| 8. | &#10003; | &#10003; | Update test matrix at [moodle-plugin-ci](https://github.com/moodlehq/moodle-plugin-ci/) GHA and Travis tests workflows. For Major release also update `*.dist.yml` templates to reflect current major branch. | Integration Team / moodle-plugin-ci maintainer |
```

Comment:
Hi, @kabalin. Do we need to perform this task during minor releases, too?

Reply:
Hi Jun, Eloy bumped test matrix in the past after minor releases, e.g. https://github.com/moodlehq/moodle-plugin-ci/commit/bce47eda9917bd5557686518a4d37224399ccfaf, but at the same time it is not critical as this is just a test matrix update, I would classify it as post-release housekeeping task (it will likely pass as main is always tested and any problem would be spotted earlier). On the other hand, I don't know if this type of change fits release checklist, thus the proposal to include it, so leaving to you to accept only major change part of it or both :)

Reply:
Thanks, @kabalin. The addition of the 404 branch to the test matrix seems to have been done by Eloy when 4.4 was released, so it coincides with the major release cycle. Thus, perhaps adding this task only for major releases is sufficient. Thanks!